File: ROME/CD/Chapter_1/settingAndSwapping.java
public class settingAndSwapping {
  static String number1="42";
  static String myName = "Jim";

  public static void main(String args[]) {

    settingAndSwapping newObj = new settingAndSwapping();
    newObj.swap();
    System.out.println("Number = " + number1 + "\n" + "Name = " + myName);
    
  }
  
  public void swap() {
    String temp = number1;
    number1 = myName;
    myName = temp;
  }
}

File: ROME/CD/Chapter_1/settingAndSwapping.js
// Setting and Swapping

var number1 = 42;
var myName = "Jojo";

function swapping() {
  var temp = number1;
  number1 = myName;
  myName = temp;
  return;
}

swapping();

console.log(number1);
console.log(myName);